mathics/builtin/box/compilation.py

```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+"""
+Boxing Routines for compiled code
+"""
```

Comment:
The final period is left off since that seems what is done elsewhere.

Reply:
The disappearing period in formatting is a separate issue that needs addressing.